backend/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -32,14 +62,57 @@ export const ApplicationPage: React.FunctionComponent = () => {
         <React.Fragment>
             <PageSection>
                 <Title headingLevel="h1"><Breadcrumb>
-                    <BreadcrumbItem to='#'> Red Hat Enterprise Linux </BreadcrumbItem>
+                    <BreadcrumbItem to='#'> </BreadcrumbItem>
                     <BreadcrumbItem to='#' isActive> { (applicationTypesQuery.loading || applicationTypesQuery.payload?.status !== 200) ?
                         <Spinner /> : applicationTypesQuery.payload.value.displayName } </BreadcrumbItem>
                 </Breadcrumb></Title>
                 <TableComposable
                     aria-label="Event types table"
                 >
                     <Thead>
+                        <Toolbar>
+                            <ToolbarContent>
+                                <ToolbarItem>
+                                    <Button variant='primary' type='submit' onClick={ toggle }> Create Event Type </Button>
+                                    <Modal
+                                        variant={ ModalVariant.medium }
+                                        title='Create Event Type'
+                                        isOpen={ isOpen }
+                                        onClose={ toggle }
+                                    ><Form isHorizontal><FormGroup label='Application Id' fieldId='application-id'
+                                            helperText='This is the id of the application'>
+                                            <TextInput
+                                                type='text'
+                                                value= { applicationId }
+                                                id='application-id' /></FormGroup>
```

Comment:
I don't think we really need to show the applicationId here, We can show in the title something like: `Create Event Type for ${the-bundle}` instead

Reply:
would it be for the bundle or for the application?

Reply:
The application, yes - my bad :)